packages/client/src/pages/ContactUs/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import {FONTS} from '../../constants';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import Button from '../../components/Button';
+import Container from '@material-ui/core/Container';
+
+const Wrapper = styled.div`
+  text-align: center;
+`;
+
+  const Title = styled.h1`
+  padding-top: 50px;
+  font-size: max(1.5vw, 32px);
+  font-weight: 400;
+  font-family: ${FONTS.font2};
+  left: calc(50% - 267px/2 + 0.5px)
```

Comment:
what's up with 267px? I'm not sure if this will scale to different screen sizes.